pkg/apis/garden/types.go

```diff
@@ -958,6 +958,8 @@ type Worker struct {
 	Labels map[string]string
 	// Taints is a list of taints for all the `Node` objects in this worker pool.
 	Taints []corev1.Taint
+	// Kubelet contains configuration settings for the kubelet.
+	Kubelet *KubeletConfig
```

Comment:
Can you please create a section `Kubernetes` and move the `kubeProxy` settings to it as well (kube-proxy and kubelet are both worker pool specific).

Reply:
What about the existing Kubernetes struct  that contains Kubernetes settings for the whole Shoot? 
Or should we put the kubeProxy struct also just flat in the Worker struct and use the settings in the spec.Kubernetes.kubeProxy as the default?

Reply:
As discussed: kubeProxy settings are not being touched as this would require more work that goes beyond this PR.